weldx/asdf/schemas/weldx.bam.de/weldx/core/geometry/point_cloud-1.0.0.yaml

```diff
@@ -0,0 +1,61 @@
+%YAML 1.1
+---
+$schema: "http://stsci.edu/schemas/yaml-schema/draft-01"
+id: "http://weldx.bam.de/weldx/core/geometry/point_cloud-1.0.0"
+tag: "tag:weldx.bam.de:weldx/core/geometry/point_cloud-1.0.0"
+
+title: |
+  Schema for a point cloud.
+description: |
+  This schema describes a collection of points in 3d space.
+
+examples:
+  -
+    - A simple PointCloud with triangulation
+    - |
+      !<tag:weldx.bam.de:weldx/core/geometry/point_cloud-1.0.0>
+        coordinates: !core/ndarray-1.0.0
+          data:
+          - [0.0, 0.0, 0.0]
+          - [1.0, 0.0, 0.0]
+          - [1.0, 1.0, 0.0]
+          - [0.0, 1.0, 0.0]
+          datatype: float64
+          shape: [4, 3]
+        triangles: !core/ndarray-1.0.0
+          data:
+          - [0, 1, 2]
+          - [2, 3, 0]
+          datatype: uint32
+          shape: [2, 3]
+
+type: object
+properties:
+  coordinates:
+    description: |
+      The coordinates of the points.
+    tag: tag:stsci.edu:asdf/core/ndarray-1.0.0
+    properties:
+      datatype:
+        type: string
+        enum: [float32, float64]
+    wx_shape: [(n), 3]
+  attributes:
+    description: |
+      An arbitrary set of attributes. For example, normals, colors or measured point data can be stored using this
+      field. Each added attribute is an array with a number of data points equal to the number of points.
+    type: object
+  triangles:
+    description: |
+      An array of index triplets that specify which points form a triangle
+    tag: tag:stsci.edu:asdf/core/ndarray-1.0.0
+    properties:
+      datatype:
+        type: string
+        enum: [uint32, uint64]
+    wx_shape: [(i), 3]
```

Comment:
Think that `[(:), 3]` would be enough because we probably don't need data that is stored per triangle. But we can leave it as is since it doesn't hurt.

Reply:
ah yes, no need to use `i` (or any letter for that matter) here you`re right